non-oss/pharos_pro/addons/kontena-stats/resources/02-configmap.yml

```diff
@@ -0,0 +1,227 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-config
+  namespace: kontena-stats
+data:
+  prometheus.yaml: |-
+    # Global config
+    global:
+      scrape_interval: 15s
+
+    # AlertManager
+    alerting:
```

Comment:
I can see some warning messages on the logs since alertmanager deployment is missing. Should we remove this or is it safe to ignore the warning?

Reply:
Good catch, this should be configurable.